app/controllers/api/v1/rents_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
 
       def create
         rent = @current_user.rents.create!(rent_params)
+        RentMailer.new_rent_send(rent.id).deliver_later if rent
```

Comment:
You don't need the validation of  rent, with  the `create!`, you are making  sure the  creation is a success

Reply:
Done